web/lib/api/api.ts

```diff
@@ -312,15 +312,29 @@ export const updateMarket = curriedAPI('market/:contractId/update')
 
 export const updateUser = curriedAPI('me/update')
 
-export function banUser(params: { userId: string; unban?: boolean }) {
-  return call(getApiUrl('ban-user'), 'POST', params)
+export function banUserFromPosting(params: {
```

Comment:
This isn't how we create api endpoints anymore, but I guess this is okay. In the future you should use the `schema.ts`